bin/main.ml

```diff
@@ -1,11 +1,12 @@
 open Cmdliner
 open Prototype
 
-let lint pkg local_repo_dir =
+let lint pkg newly_published local_repo_dir =
   match local_repo_dir with
   | Some d ->
       print_endline @@ Printf.sprintf "Linting %s in %s ..." pkg d;
-      Lint.run_lint pkg d
+      (* TODO: Automatically detect newly published *)
```

Comment:
Good idea here. Might make sense to add an issue to track this?

Reply:
Sure.

Reply:
https://github.com/tarides/infrastructure/issues/343